lib/krikri/harvesters/fedora_harvester.rb

```diff
@@ -0,0 +1,158 @@
+require 'uri'
+require 'async_uri_getter'
+
+module Krikri::Harvesters
+  ##
+  # A harvester implementation for Fedora 3
```

Comment:
Beauty.  I'll let James address the couple of issues raised that remain, and then we'll move them all over.